docs/concepts.md

```diff
@@ -192,10 +192,12 @@ example when a Node sends a value over a Channel. Given two labels `L_a` and
 `L_a` can be sent to a destination labeled `L_b`. The "flows to" operator
 compares both confidentiality and integrity:
 
-- the **confidentiality** check ensures that the destination is permitted to
-  view any secrets that may have influenced the value
-- the **integrity** check ensures that the value sent is trusted by the
-  destination
+- the **confidentiality** check protects the source from leaking secret data. It
+  does this by checking that the destination is permitted to view any secrets
+  that may have influenced values sent by the source.
+- the **integrity** check protects the destination from receiving potentially
+  malicious or inaccurate data. It does this by checking that the source is
+  trusted by the destination.
```

Comment:
In [the slack thread](https://project-oak.slack.com/archives/CC03DDX3M/p1593173725212500), @benlaurie mentioned that he thinks integrity has other uses too, such as the prevention of covert channels. 

On a higher level I think that stating a high level purpose of each check here is still a good idea. It supports the reader in building a mental model of the IFC, even if the stated purpose is not 100% exhaustive.